docs/node-operation/itn-manual.md

```diff
@@ -0,0 +1,210 @@
+
+### Phase 4: Decentralized publication & Mainnet soft launch
+
+The final phase will see all validators with the functional ability to
+participate in collection, validation and publication on-chain (when nominated).
+
+During this phase, the consensus mechanism will be expanded so as to select a
+quorum of validators with approved data in order to then elect one to publish
+their Fact Statement on-chain and be rewarded for it.
+
+After this phase has been tested successfully, we will fully move to Mainnet.
+
+## Joining the Incentivized Testnet
+
+Commitment will key for those participating in the ITN. We expect participants
+to engage earnestly with the process, dedicating the necessary time and energy.
+This will include actively providing feedback that will be instrumental in
+refining and strengthening the Mainnet protocol.
+
+To join the ITN, participants must meet the following:
+
+### Validator License
+
+Initially, participants may only be required to hold a validator license on a
+Mainnet wallet. However, as we progress through the more advanced phases of the
+ITN, participants will also need one on Testnet.
+
+Holding a validator license on Mainnet will allow participants to mint one on
+Testnet.
+
+```text
+NB
+ITN rewards cannot be earned without a valid license.
+```
+
+### 500,000 FACT deposit
+
+Participants need to maintain this balance in their Mainnet wallet, which will
+later need to be secured in a smart contract on the Testnet with tFACT
+(Testnet FACT).
+
+### Hardware and Infrastructure
+
+During [Phase 1](#phase-1-data-collection-for-cexes) of the ITN, the hardware
+requirements will be minimal to handle the tasks involved. Participants should
+expect to meet the following:
+
+* A broadband internet connection
+* 2GB of RAM
+* A single CPU
+
+However, as we progress through the ITN, the complexity and demands will
+increase. Because of this, Participants should consult our
+[Hardware Requirements](technical-requirements#hardware) section while also
+anticipating that these requirements will change as development, and ITN phases,
+progress.
+
+### Software requirements
+
+Starting with [Phase 2](#phase-2-incorporating-dex-data), participants will be
+expected to operate more taxing software (eg a Cardano node, Ogmios, etc.).
+
+Participants should consult our
+[Software Requirements](technical-requirements#software) section while also
+anticipating that these requirements will change as development, and ITN phases,
+progress.
+
+### Knowledge requirements
+
+Participants must possess the skills to operate a Linux server and install
+various software packages, including those provided by Orcfax. Additionally,
+participants should be capable of updating both the operating system and any
+installed software packages to maintain security and functionality.
+
+While it is not a requirement at the outset, having an understanding of the
+Cardano node and its underlying protocols is highly recommended to facilitate a
```

Comment:
i wonder if this is more about an understanding of operating a cardano node?

we will be doing our utmost to wrap all this functionality and making the validators as appliance like as possible. (this is for our own sakes not just the ITN participants).